framework/dependencies.yml

```diff
@@ -69,8 +68,9 @@ require:    &allDependencies
     - org.javassist -> javassist 3.28.0-GA
     - io.netty -> netty 3.10.6.Final
     - org.postgresql -> postgresql 42.3.1
-    - org.slf4j -> slf4j-api 1.7.33
```

Comment:
@tazmaniax because reload4j is not verry common and a potential future issue I would personally recommend using the log4j2 bridge (https://logging.apache.org/log4j/2.x/manual/migration.html#Log4j1.2Bridge). I have it working in Play using the following:
    - org.apache.logging.log4j -> log4j-1.2-api 2.17.0
    - org.apache.logging.log4j -> log4j-core 2.17.0
    - org.apache.logging.log4j -> log4j-api 2.17.0

Additionnaly, we have to set the system property -Dlog4j1.compatibility=true
Maybe here is a good place: https://github.com/playframework/play1/blob/21a3156542cf36e8dac5fe9f285d8b4a3c427cce/framework/src/play/server/Server.java#L28

Reply:
The replace of Log4j with reload4j has been reverted from this PR